templates/web/base/reports/_other_districts-list.html

```diff
@@ -0,0 +1,58 @@
+[% SET show_parish_shown_class = c.req.params.show_parishes ? '' : 'js-parish-multi hidden' %]
+<h2>[% parish_text %]</h2>
+    <p class="[% show_parish_hidden_class %]">[% loc('Follow a parish link to view only reports within that parish.') %]</p>
+    <p class="[% show_parish_shown_class %]">[% loc('Select multiple parishes to view only reports within those parishes.') %]</p>
+    <p class="[% show_parish_hidden_class %]"><a href="?show_parishes=1" class="js-parish-select-multiple">[% loc('Select multiple parishes?') %]</a></p>
+    <form method="get" action="">
+    <section class="full-width">
+    <ul class="item-list item-list--wards">
+      [% FOR child IN covered.values.sort('name') %]
+        [% IF child.type == 'CPC' %]
+        <li class="item-list__item item-list--wards__item">
+          <label class="[% show_parish_shown_class %]">
+            <input type="checkbox" name="ward" value="[% child.name | html %]">
+            [% child.name %]
+          </label>
+          <a class="[% show_parish_hidden_class %]" href="[% child.url %]">[% child.name %]</a>
+        </li>
+        [% END %]
+      [% END %]
+    </ul>
+    </section>
+    <p>
+      <input class="btn [% show_parishes_shown_class %]" type="submit" value="[% loc('View parishes') %]" />
+    </p>
+    </form>
+</section>
+<section id="council_district_wards"[% UNLESS c.req.params.show_districts %] class="hidden-js"[% END %]>
+  [% SET show_district_hidden_class = c.req.params.show_districts ? 'hidden' : 'js-district-single' %]
+  [% SET show_district_shown_class = c.req.params.show_districts ? '' : 'js-district-multi hidden' %]
+  <h2>[% district_text %]</h2>
+      <p class="[% show_district_hidden_class %]">[% loc('Follow a district link to view only reports within that district.') %]</p>
+      <p class="[% show_district_shown_class %]">[% loc('Select multiple districts to view only reports within those districts.') %]</p>
+      <p class="[% show_district_hidden_class %]"><a href="?show_districts=1" class="js-district-select-multiple">[% loc('Select multiple districts?') %]</a></p>
+      <form method="get" action="">
+      <section class="full-width">
+      <ul class="item-list item-list--wards">
+        [% FOR child IN covered.values.sort('name') %]
+          [% IF child.type == 'DIW' %]
+          <li class="item-list__item item-list--wards__item">
+            <label class="[% show_district_shown_class %]">
+              <input type="checkbox" name="ward" value="[% child.name | html %]">
+              [% child.name %]
+            </label>
+            <a class="[% show_district_hidden_class %]" href="[% child.url %]">[% child.name %]</a>
+          </li>
+          [% END %]
+        [% END %]
+      </ul>
+      </section>
+      <p>
+        <input class="btn [% show_district_shown_class %]" type="submit" value="[% loc('View districts') %]" />
+      </p>
+      </form>
+  </section>
+[% END %]
```

Comment:
This is a base template but the javascript is only for Oxfordshire. Perhaps this should be in an Oxfordshire-specific template for now (with appropriate try/catch where it's included). Then if/when we want to reuse this on other cobrands in the future we can see which bits can be shared.